node/nodeRunner.go

```diff
@@ -1591,9 +1567,35 @@ func (nr *nodeRunner) CreateManagedCryptoComponents(
 	return managedCryptoComponents, nil
 }
 
+// CreateArgsRunTypeComponents - creates the args for runType components
+func (nr *nodeRunner) CreateArgsRunTypeComponents(coreComponents mainFactory.CoreComponentsHandler, cryptoComponents mainFactory.CryptoComponentsHandler) (*runType.ArgsRunTypeComponents, error) {
+	initialAccounts := make([]*data.InitialAccount, 0)
```

Comment:
Duplicated code from soveriegn. We can reuse this

Reply:
this is fixed in the next [PR](https://github.com/multiversx/mx-chain-go/pull/6130)